From c6868e91438ed81257b939f0a36c1abeda207c26 Mon Sep 17 00:00:00 2001
Message-Id: <c6868e91438ed81257b939f0a36c1abeda207c26.1645111732.git.jbenc@redhat.com>
From: Jiri Benc <jbenc@redhat.com>
Date: Thu, 17 Feb 2022 16:28:06 +0100
Subject: [PATCH] Pass LDFLAGS to the linker

Signed-off-by: Jiri Benc <jbenc@redhat.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 0423f3198c09..54d7aa552f1b 100644
--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,7 @@ plotnetcfg: args.o ethtool.o frontend.o handler.o if.o label.o main.o match.o ne
 	    handlers/bridge.o handlers/master.o handlers/openvswitch.o handlers/veth.o \
 	    handlers/vlan.o \
 	    frontends/dot.o frontends/json.o
-	gcc -o $@ $+ $(libs)
+	gcc $(LDFLAGS) -o $@ $+ $(libs)
 
 args.o: args.c args.h
 ethtool.o: ethtool.c ethtool.h
-- 
2.34.1

